### PR TITLE
Update oracle docs

### PR DESCRIPTION
**Pull request by mcat115**

### CODE REVIEW

File: docs/manual/projects/resource-model-sources/oracle.md [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 To configure Authentication use this page as a reference: [Create Credential for OCI Authentication](https://docs.oracle.com/en/cloud/paas/management-cloud/logcs/create-credentials-oci-authentication.html)
 
+When setting up the node for your Oracle instance, make the compartment ID the same as the tenant ID if you are getting an error and your root tenant doesn't have any compartments.
```
Comment (jsboak) [inline docs/manual/projects/resource-model-sources/oracle.md]: @mcat115 i would put this in a "tip" block

:::tip Note
When setting up the node for your Oracle instance, make the compartment ID the same as the tenant ID if you are getting an error and your root tenant doesn't have any compartments.
:::

File: docs/manual/workflow-steps/oracle.md [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 To configure Authentication use this page as a reference: [Create Credential for OCI Authentication](https://docs.oracle.com/en/cloud/paas/management-cloud/logcs/create-credentials-oci-authentication.html)
 
+When setting up the node for your Oracle instance, make the compartment ID the same as the tenant ID if you are getting an error and your root tenant doesn't have any compartments.
```
Comment (jsboak) [inline docs/manual/workflow-steps/oracle.md]: @mcat115  i would also put this in a "tip" block

**Comment by mcat115** [comment]
that syntax didn't work, so I googled another type that I thought looked good for a tip :)

**Review by jsboak — APPROVED** [review]
LGTM 👍

**Review by fdevans — APPROVED** [review]

**Pull request merged**